<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>capy::ui::ColorPaletteArea</class>
    <widget class="QWidget" name="capy::ui::ColorPaletteArea">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="topMargin">
                <number>6</number>
            </property>
            <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Palette:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="paletteComboBox"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <item>
                                <widget class="QPushButton" name="createPaletteButton">
                                    <property name="text">
                                        <string>Create</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="removePaletteButton">
                                    <property name="text">
                                        <string>Delete</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="Line" name="line">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QTableView" name="colorTableView"/>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
